### PR TITLE
[new release] lambda-term (3.3.2)

### PR DESCRIPTION
**Pull request by kandu**
3.3.2 (2023-08-09)
------------------

* `LTerm_vi`: fix a downward-action issue when act in empty content

### CODE REVIEW

File: packages/lambda-term/lambda-term.3.3.2/opam [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+synopsis: "Terminal manipulation library for OCaml"
+description: """
+Lambda-term is a cross-platform library for manipulating the terminal. It
+provides an abstraction for keys, mouse events, colors, as well as a set of
+widgets to write curses-like applications. The main objective of lambda-term is
+to provide a higher level functional interface to terminal manipulation than,
+for example, ncurses, by providing a native OCaml interface instead of bindings
+to a C library. Lambda-term integrates with zed to provide text edition
+facilities in console applications."""
+maintainer: ["opam-devel@lists.ocaml.org"]
```
Comment (kit-ty-kate) [inline packages/lambda-term/lambda-term.3.3.2/opam:11]: This email address is for the opam developers, who are not maintaining this package. Is there any way to put yourself there instead?

Comment (kandu) [reply packages/lambda-term/lambda-term.3.3.2/opam:11]: That would be fine :)

**Comment by mseri** [comment]
Are pyast failures related to this?

**Comment by kandu** [comment]
This is the change of this bugfix release https://github.com/ocaml-community/lambda-term/pull/116/commits/910bba1ac36329f6b605d2bc54dc4b2aee756eb0
API or internal logic is untouched. So it is unlikely to cause new build failure with revdeps.

**Comment by mseri** [comment]
Thanks

**Pull request merged**